WordPressPCL/WordPressClient.cs

```diff
@@ -146,6 +146,38 @@ public WordPressClient(string uri, string defaultPath = "wp/v2/")
             CustomRequest = new CustomRequest(ref _httpHelper);
         }
 
+        /// <summary>
+        ///     The WordPressClient holds all connection infos and provides methods to call WordPress APIs.
+        /// </summary>
+        /// <param name="httpClient">URI for WordPress API endpoint, e.g. "http://demo.wp-api.org/wp-json/"</param>
+        public WordPressClient(HttpClient httpClient)
+        {
+            if (httpClient == null)
+            {
+                throw new ArgumentNullException(nameof(httpClient));
+            }
+            string uri = httpClient.BaseAddress.ToString();
+            if (!uri.EndsWith("/", StringComparison.Ordinal))
+            {
+                uri += "/";
+            }
+            WordPressUri = uri;
+            _defaultPath = string.Empty;
+
+            _httpHelper = new HttpHelper(httpClient);
+            Posts = new Posts(ref _httpHelper, _defaultPath);
```

Comment:
Cannot happens, since you have properties declared as `Post {get;}`  which allow initialisation only inside constructor. so moving to separate function would require change to `{ get; private set; }` which probably not something you really want. But that's up to you

Reply:
yea I've seen the build issue. I guess this could be resolved like this https://stackoverflow.com/questions/4009013/call-one-constructor-from-another
but we can do this at a later point as well

Reply:
Anything else to change, or this is ready to be accepted?